pkg/apis/features/features.go

```diff
@@ -0,0 +1,21 @@
+package features
+
+import (
+	"context"
+
+	tektonconfig "github.com/tektoncd/pipeline/pkg/apis/config"
+)
+
+// SetFeatureFlag sets the feature flag for Tekton parsers in the context.
+func SetFeatureFlag(ctx context.Context) context.Context {
+	featureFlags, _ := tektonconfig.NewFeatureFlagsFromMap(map[string]string{
+		"enable-tekton-oci-bundles": "true",
+	})
+	cfg := &tektonconfig.Config{
+		Defaults: &tektonconfig.Defaults{
+			DefaultTimeoutMinutes: 60,
```

Comment:
Why do you need that one ? 🤔

Reply:
ah yeah that was during my test, doesn't matter much tho...